bin/test.sh

```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
-#
-# Simple smoke test for the decode-dist tool.  This will fail if your machine
-# doesn't have the right R libraries.
-#
-# Usage:
-#   ./test.sh <function name>
-#
-# Example:
-#   ./test.sh write-assoc-testdata       # write data needed for R and C++ tests
-#   ./test.sh decode-assoc-R-smoke       # test pure R implementation
-#   ./test.sh decode-assoc-cpp-smoke     # test with analysis/cpp/fast_em.cc
-#   ./test.sh decode-assoc-cpp-converge  # run for longer with C++
-#   ./test.sh decode-assoc-tensorflow
+usage() {
+echo "
```

Comment:
Should usage() be called by default if ./test.sh is called without parameters or with "--help"? Is this a standard convention?